frontend/build-and-copy.sh

```diff
@@ -2,15 +2,19 @@
 
 set -e
 
-FROM=./result
+RESULT=./result
+OUT=./out
 TO=./../api/src/treestatus_api/static/ui
 
-rm -f $FROM
-nix-build -o $FROM
+rm -f $RESULT
+rm -rf $OUT
+mkdir $OUT
+docker container rm treestatus || :
+docker run --name treestatus -v $(pwd):/src nixos/nix sh -c "cd /src && nix-build -o $RESULT"
+# pipe through tar to avoid permission denied issues
+docker container cp treestatus:$(readlink -f $RESULT)/ - | tar -C $OUT --strip-components=1 -x
+sudo chown $(id -un): -R $OUT
```

Comment:
Some sort of message prior to the password prompt would be helpful to avoid 'huh, what's this for?' reactions. FWIW the chown wasn't required here because the files were already owned by my user.

The `-R` arg needs to be immediately after `chown` on mac.

Reply:
Probably I can remove chown, because there is no copy anymore, piping through tar fixes all the things :)